### ISSUE TITLE
New feature: Executive summary of datasets under each topic

### ISSUE THREAD

**Issue by strehan1** [GSA/datagov-wptheme#686]
Data.gov is a wonderful index of datasets from difference government sources. However, the site navigation seems to be more suited for someone who knows what he/she is looking for exactly. To make the website more user friendly and increase accessibility and discoverability of data for the general public, I think we need a feature that provides a quick summary of the data sets under each topic by popular tags/issue areas. 

For instance, when I open the Climate data catalog (http://catalog.data.gov/dataset?groups=climate5434&#topic=climate_navigation), I see 740 datasets with some categorization filtering and 2 line descriptions. But within each categorization (for e.g., Arctic), there are again 200+ datasets. To make a citizen interested in knowing more about, say, climatic changes, it would be helpful to have a summary on top which says something like: 
To explore more about Arctic climatic changes see xx, yy datasets; details on water table are on xx, yy; etc. The datasets shown as hyperlinked icons could be selected based on popularity and relevance (much like a Google Page Rank or Amazon). 

This feature, I believe will help make data.gov more accessible for the common (non/low technical) public serving its objective of increased participation. If someone has explored this line of thinking and already knows the right approach for navigation which provides similar results, please guide me to the same.

**Comment by benzevgreen** [comment]
+1 to this

In addition, I think there is significant value to be added to the site by providing a) potential uses of data that would contribute to important issues, and b) case studies of past projects that have used resources on data.gov. In some cases, I have found that civic tech hackers and other citizens want to produce something useful with data.gov but struggle to find the right problem to solve or the best data to use. Other platforms such as the Apple App Store provide resources for developers to help them make the best use of the resources available. Adding similar features to data.gov would be a large step forward toward making it into a platform on which more people can create value.

**Comment by ywthls** [comment]
Agreed--data.gov is a great hub for accessing data on a wide array of topics, but its organization leaves something to be desired. If the point is to make data available for the public, the explanations should be written in language that a non-expert member of the public could understand. In the site's current state, it takes a lot of sifting and reading to find data sets that have potential applications to the problem being researched. 

One possible solution to this problem is to organize the summary into several categories. Instead of uploading a long, technical paragraph as the summary, uploaders would be directed to answer the following questions:
(1) Who collected the data?
(2) What was the data collected from?
(3) How was the data collected?
(4) Why was the data collected?
(5) What categories of information does this data offer?

Users would be able to read the uploader's answers to these questions when they click through to the data set's landing page. With these questions outlined for every data set, users will know how the summaries are organized and will be able to more quickly read through them and find the relevant data sets for their research. 

A screenshot of the data sets might also be useful, as this would enable users to more quickly see if a given data set is applicable to their research. Clicking through the links and downloading data sets takes a while, especially because users have to navigate through multiple websites' download systems. Providing a screenshot would allow users to skip these steps and would lower the barrier to entry for people who want to use the data for entrepreneurial solutions.
